Source/Applications/SystemCenterNotification/Scripts/TSX/EmailTypes/ByEmailType.tsx

```diff
@@ -109,56 +111,103 @@ const ByEmailType = (props: IProps) => {
-                                    Add Email
-                                </button>
-                            </form>
-                        </fieldset>
-                    </li>
-                </SearchBar>
-                <div style={{ width: '100%', height: 'calc( 100% - 136px)' }}>
-                    <Table<EmailType>
-                        cols={[
-                            { key: 'Name', field: 'Name', label: 'Name', headerStyle: { width: '30%' }, rowStyle: { width: '30%' } },
-                            { key: 'MaxDelay', field: 'MaxDelay', label: 'Maximum Delay (s)', headerStyle: { width: '15%' }, rowStyle: { width: '15%' } },
-                            { key: 'MinDelay', field: 'MinDelay', label: 'Minimum Delay (s)', headerStyle: { width: '15%' }, rowStyle: { width: '15%' } },
-                            { key: 'SMS', field: 'SMS', label: 'Text Message', headerStyle: { width: '15%' }, rowStyle: { width: '15%' }, content: (item) => item.SMS ? HeavyCheckMark : CrossMark },
-                            { key: 'ShowSubscription', field: 'ShowSubscription', label: 'Self Subscription', headerStyle: { width: '15%' }, rowStyle: { width: '15%' }, content: (item) => item.ShowSubscription ? HeavyCheckMark : CrossMark },
-                            { key: 'RequireApproval', field: 'RequireApproval', label: 'Requires Approval', headerStyle: { width: '10%' }, rowStyle: { width: '10%' }, content: (item) => item.RequireApproval ? HeavyCheckMark : CrossMark },
-                            { key: 'scroll', label: '', headerStyle: { width: 17, padding: 0 }, rowStyle: { width: 0, padding: 0 } },
-                        ]}
-                        tableClass="table table-hover"
-                        data={data}
-                        sortKey={sortField}
-                        ascending={asc}
-                        onSort={(d) => {
-                            if (d.colKey === 'scroll' || d.colKey === 'undefined')
-                                return
-                            if (d.colField === sortField)
-                                dispatch(EmailTypeSlice.Sort({ SortField: sortField, Ascending: asc }));
-                            else
-                                dispatch(EmailTypeSlice.Sort({ SortField: d.colField, Ascending: true }));
+            <div className="row">
+                <div className="col">
+                    <SearchBar<EmailType> CollumnList={searchFields}
+                        SetFilter={(flds) => dispatch(EmailTypeSlice.DBSearch({ filter: flds }))}
+                        Direction={'left'} defaultCollumn={{ key: 'Name', label: 'Name', type: 'string', isPivotField: false }} Width={'50%'} Label={'Search'}
+                        ShowLoading={searchStatus === 'loading'} ResultNote={searchStatus === 'error' ? 'Could not complete Search' : 'Found ' + data.length + ' Event Notification(s)'}
+                        GetEnum={() => {
+                            return () => { }
```

Comment:
I don't think you need to return a cleanup function if there is nothing to clean up